src/components/CommentCard.jsx

```diff
@@ -1,21 +1,37 @@
 import { useEffect, useState } from "react";
-import { getComments } from "../api";
+import { getComments, deleteComment } from "../api";
 import Carousel from "react-bootstrap/Carousel";
 import AddComment from "./AddComment";
 
 function CommentCard({ article_id , commentCount}) {
   const [comments, setComments] = useState([]);
   const [index, setIndex] = useState(1);
+  const [error, setError] = useState();
```

Comment:
A good practice is to keep state values the same data type, as the `error` is going to be a _string_, initialise it with `""`